SetupScript.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+(cd /opt/ && git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git)
+(cd /opt/Epsilon-Raspberry/primary/ && cp xorg.conf /etc/X11/ && cp config.txt /boot/)
+
+#install rabbit-mq
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | tee /etc/apt/sources.list.d/rabbitmq.list && apt-get update && apt-get install rabbitmq-server
+apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev software-properties-common build-essential mesa-common-dev libgl1-mesa-dev
+
+cd /tmp/
+if ls /usr/local/lib/librabbitmq.* 1> /dev/null 2>&1 ;
```

Comment:
This is a more pragmatic way to check if something is empty. 

https://serverfault.com/questions/7503/how-to-determine-if-a-bash-variable-is-empty

Also I would recommend doing this for formatting if else statements in bash.

`if [ condition]; then
    soething
else
    blah blah
fi